src/ripple/rpc/impl/TransactionSign.cpp

```diff
@@ -561,12 +561,11 @@ transactionConstructImpl(
     // Turn the passed in STTx into a Transaction.
     Transaction::pointer tpTrans;
     {
-        std::string reason;
-        tpTrans = std::make_shared<Transaction>(stpTrans, reason, app);
+        tpTrans = std::make_shared<Transaction>(stpTrans);
         if (tpTrans->getStatus() != NEW)
         {
-            ret.first = RPC::make_error(
-                rpcINTERNAL, "Unable to construct transaction: " + reason);
+            ret.first =
+                RPC::make_error(rpcINTERNAL, "Unable to construct transaction");
             return ret;
         }
```

Comment:
This block should be impossible to reach now. It can probably be safely replaced with `assert(tpTrans->getStatus() == NEW);`